sdks/python/setup.py

```diff
@@ -237,6 +237,7 @@ def get_portability_package_data():
         'regex>=2020.6.8',
         'requests>=2.24.0,<3.0.0',
         'typing-extensions>=3.7.0',
+        'regex>=2022.3.2',
```

Comment:
Why do we need this now?

Reply:
Otherwise LGTM

Reply:
The original Python Cron Precommit, install some dependencies before run the tests. 
At the moment of running this tests isolated the dependency creates an error because regex version.
The solution is added this regex version in setup.py  to provide it to the tests in the gradle job

Reply:
Shouldn't adding it to the test_requires section be enough? Adding it to the list of install_requires dependencies seems unnecessary to me.

Reply:
Done, regex delete from install_requires. 
Only in test